support/Environments/wheeler_gcc.sh

```diff
@@ -59,6 +60,7 @@ spectre_load_modules() {
     module load hdf5/1.12.2
     module load libbacktrace/1.0
     module load spec-exporter/2023-07
+    module load paraview/5.11.1-osmesa
```

Comment:
Our [sxs docs for wheeler](https://github.com/sxs-collaboration/WelcomeToSXS/wiki/Wheeler#paraview-for-visualizations) give instructions using `5.10.1`. Maybe we should use that?

Reply:
Those use an old Python 3.7